transport/http_json/swagger_json.go

```diff
@@ -146,6 +147,10 @@ func generateSwaggerJSON(hm *gorpc.HandlersManager) ([]byte, error) {
 				operation.Description += ".<br/>Handler caches response."
 			}
 
+			if v.ExtraData != nil {
+				operation.Description += fmt.Sprintf("\n<!-- <extra_data>%v</extra_data> -->", v.ExtraData)
```

Comment:
Are you sure that extra data have to be in comment?
I think you should add callback that will add extra information to description